editor/editor_inspector.h

```diff
@@ -231,6 +231,9 @@ class EditorInspectorSection : public Container {
 	Color bg_color;
 	bool foldable;
 
+	Timer *dropping_unfold_timer;
```

Comment:
=  nullptr.

Reply:
Sorry, I don't understand. `memnew(Timer)` is used in the constructor. Why does it need `= nullptr`? That is not used anywhere else in the codebase where a `Timer*` is defined.

Reply:
https://github.com/godotengine/godot/blob/b2096ba53eb422ae2fbf8b49cf1f4806878127fb/modules/mono/csharp_script.h#L364 Here's an example of:

```
Timer *dropping_unfold_timer = memnew(Timer);
```

Reply:
Yeah so that's in the header, I'm saying the timer is initialised in the constructor:

https://github.com/godotengine/godot/pull/41138/files#diff-554c9c97362d5a6f63b7244253542ffeR1312

I'm just wondering why this needs to specify `= nullptr` when I can't seem to find that happening anywhere else in the codebase. For example:

https://github.com/godotengine/godot/blob/d3b5c0948c943b3fbcb4b5322262c59c92abfa83/scene/gui/spin_box.h#L44

It is then initialised in the constructor:

https://github.com/godotengine/godot/blob/d3b5c0948c943b3fbcb4b5322262c59c92abfa83/scene/gui/spin_box.cpp#L272-L274

Reply:
I don't remember exactly, but every so often Akien runs a script that moves the construction stuff to initialization.

Reply:
Ah ok, no worries. I was just wondering if there was some convention as I couldn't see other cases of this being used.